.github/workflows/jepsen.yml

```diff
@@ -24,23 +33,24 @@ on:
         required: true
         default: 60
       tendermintUrl:
-        description: 'Where to grab the Tendermint binary (linux/amd64 tarball)?'
+        description: 'Where to grab the Tendermint tarball (w/ linux/amd64 binary)?'
         required: true
         default: 'https://github.com/melekes/katas/releases/download/0.2.0/tendermint.tar.gz'
```

Comment:
Would you want this to be under the tendermint org or are you fine with this being in your github repo?

Reply:
I really wish, but until https://github.com/tendermint/tendermint/pull/5772 lands (note: it's not a part of v0.34 because it breaks CLI), there's no way other than using a custom build.